src/customers-container/customers-store.js

```diff
@@ -73,8 +73,11 @@ class CustomersStore {
         customerInfo.url,
         ''
       );
-      await this.makeHttpRequest(UPDATE_CUSTOMER_QUERY);
-      this.fetchCustomers();
+      const response = await this.makeHttpRequest(UPDATE_CUSTOMER_QUERY);
+      if (response.updateCustomer) {
+        alert('update customer successfully');
+        this.fetchCustomers();
+      }
     } catch (error) {
       console.log('cant save customer');
```

Comment:
at least use `console.warn(error)` for stack trace

Reply:
fixed